### ISSUE TITLE
Picking up unsupported fields 

### ISSUE THREAD

**Issue by NickCH-K** [brechtm/citeproc-py#86]
I will admit to being a naif so this may be a silly question, but is there a good reason why citeproc-py's parser won't pick up attributes that aren't on the official list?

Sure, unsupported fields won't have a way of being used properly with a CSL. But if you had other uses for a parsed BibTeX file, or if you needed to fix some keys before using it with CSL (say you've got a file where the key is pubyear instead of year, or someone sent you a Better BibTeX file, which doesn't play nice with citeproc-py at all) it would be nice for it to pick up everything and just store it as unprocessed dict items in the Reference dict. 

For my purposes I just built a secondary parser to get all this stuff and stick it in each Reference and it seems to work fine, so it shouldn't get in the way of anything. But I haven't done a lot of testing yet.

**Comment by brechtm** [comment]
Yes, I agree that is it unnecessary to abort on encountering an unknown key in a BibTeX file. It would be better to simply output a warning instead and continue parsing.

**Comment by NickCH-K** [comment]
Doesn't it already do that? I was wondering why it leaves the unrecognizable types out of the Reference dict, rather than adding them in and just ignoring them for CSL purposes.

**Comment by brechtm** [comment]
Oops, that could very well be. I haven't checked the code. Pull requests are welcome :-)

**Comment by NickCH-K** [comment]
I might just do that then!
